src/DataMiner/SLManagedAutomation/UIBuilder.cs

```diff
@@ -74,35 +74,59 @@ public class UIBuilder
 		/// </example>
 		public int Height { get; set; }
 
-		/// <summary>
-		/// Gets or sets the maximum height (in pixels) of the dialog box.
-		/// </summary>
-		/// <value>The minimum height (in pixels) of the dialog box.</value>
-		/// <remarks>
-		/// <note type="note">Interactive Automation scripts resize their width and height depending on which properties of the script are filled in (UIBuilder.Height, .Width, .MinHeight, .MinWidth). If nothing is defined in the script, a default width and height of 650px by 650px will be applied.</note>
-		/// </remarks>
-		/// <example>
-		/// <code>
-		/// UIBuilder uib = new UIBuilder();
-		/// uib.MaxHeight = 400;
-		/// </code>
-		/// </example>
-		public int MaxHeight { get; set; }
+        /// <summary>
+        /// Gets or sets the column with the highest number that has one or more components.
+        /// </summary>
+        /// <value>The column with the highest number that has one or more components.</value>
+        /// <example>
+        /// <code>
+        /// UIBuilder uib = new UIBuilder();
+        /// uib.MaxCol = 2;
+        /// </code>
+        /// </example>
+        public int MaxCol { get; set; }
```

Comment:
Can you explain the purpose of this property?

Reply:
the purpose of this property is to get or set the column with the highest number that has one or more components.